.env

```diff
@@ -29,7 +30,23 @@ OKP4_COOKIE_POLICY_URL='https://okp4.network/cookie-policy'
 OKP4_GITHUB_OKP4D_URL='https://github.com/okp4/okp4d'
 OKP4_GITHUB_CONTRACTS_URL='https://github.com/okp4/contracts'
 OKP4_GITHUB_AWESOME_URL='https://github.com/okp4/awesome'
+OKP4_GITHUB_OKP4D_RELEASES_URL='https://github.com/okp4/okp4d/releases'
+OKP4_GITHUB_OBJECTARIUM_URL='https://github.com/okp4/contracts/tree/main/contracts/okp4-objectarium'
```

Comment:
@ccamel
So what should we do?
Could we merge this PR with this url and then make the changes when the project will be released?
Because, while we are waiting on this PR, the product team could not test it

Reply:
@houssemmimoun27 

Have you handled the links provided by @amimart ?

Reply:
Great, so let's wait for the review of @ccamel